### ISSUE TITLE
y_true negative dimensions

### ISSUE THREAD

**Issue by ahmarz** [mokemokechicken/keras_BEGAN#1]
when trying to use y_true in the loss function i receive the following error : 

InvalidArgumentError (see above for traceback): Shape [-1,-1,-1,-1] has negative dimensions
[[Node: decoder/FinalConv_target = Placeholderdtype=DT_FLOAT, shape=[?,?,?,?], _device="/job:localhost/replica:0/task:0/gpu:0"]]

appreciate your help

**Comment by mokemokechicken** [comment]
Hi, @ahmelshi 

y_true is not passed to the loss function in my implementation.
The error might be fixed if change this.

```
loss_generator = generator.train_on_batch(in_x1, np.zeros_like(in_x2))  # y_true is meaningless
```
https://github.com/mokemokechicken/keras_BEGAN/blob/master/src/began/training.py#L81
